### ISSUE TITLE
support reading `r:streamContent`

### ISSUE THREAD

**Issue by gavinr** [StevenLooman/python-didl-lite#5]
Thanks for the library!

My didl-lite string looks like:

```
<?xml version="1.0" encoding="UTF-8"?>
<DIDL-Lite xmlns="urn:schemas-upnp-org:metadata-1-0/DIDL-Lite/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:r="urn:schemas-rinconnetworks-com:metadata-1-0/" xmlns:upnp="urn:schemas-upnp-org:metadata-1-0/upnp/">
   <item id="-1" parentID="-1" restricted="true">
      <res protocolInfo="sonos.com-http:*:application/x-mpegURL:*">x-sonosapi-hls:live%3asymphonyhall?sid=37&amp;flags=8480&amp;sn=4</res>
      <r:streamContent>BR P|TYPE=SNG|TITLE Classical|ARTIST fb.com/SymphonyHall|ALBUM</r:streamContent>
      <dc:title>x-sonosapi-hls:live:symphonyhall?sid=37&amp;flags=8480&amp;sn=4</dc:title>
      <upnp:class>object.item.audioItem.musicTrack</upnp:class>
   </item>
</DIDL-Lite>
```

I see that this library grabs the `<res>` - https://github.com/StevenLooman/python-didl-lite/blob/a25fa2bd47f1676a99ec08f8436360ee80f94d58/didl_lite/didl_lite.py#L108 ... but is there a way to grab the `<r:streamContent>` conent?

**Comment by StevenLooman** [comment]
Thanks for your issue!

There seems no way to get the additional contents. Would it be enough for you that the XML element itself is accessible?

**Comment by gavinr** [comment]
> Would it be enough for you that the XML element itself is accessible

I do not know enough about didl-lite to know what would be best, but that seems reasonable to me.

**Comment by StevenLooman** [comment]
@gavinr See the PR, will that work for you?

**Comment by gavinr** [comment]
@StevenLooman I think so, thank you!

**Comment by StevenLooman** [comment]
I've released version `1.2.5`. Hope this helps!
